vault/resource_kmip_secret_role.go

```diff
@@ -0,0 +1,287 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var kmipRoleAPIBooleanFields = []string{
+	"operation_activate",
```

Comment:
I wonder if we should factor these field names out to be constants?

E.g.

`fieldOperationDestroy`

Reply:
Added in 3cba7a6